scanner/indexer/indexer.go

```diff
@@ -32,7 +34,8 @@ type Indexer interface {
 	Close(context.Context) error
 }
 
-type indexerImpl struct {
+// localIndexer is the Indexer implementation that runs libindex locally.
+type localIndexer struct {
```

Comment:
curious why the name change

Reply:
Because this is the Indexer implementation that runs libindex locally.